projects/ngrx.io/content/guide/data/entity-actions.md

```diff
@@ -0,0 +1,269 @@
+# Entity Actions
+
+The [`EntityCollectionService`](guide/data/entity-services) dispatches an `EntityAction` to the _NgRx store_ when you call one of its commands to query or update entities in a cached collection.
+
+## _Action_ and _EntityAction_
+
+A vanilla
+[_NgRx `Action`_](guide/store/actions) is a message.
+The message describes an operation that can change state in the _store_.
+
+The _action_'s `type` identifies the operation.
+It's optional `payload` carries the message data necessary to perform the operation.
+
+An `EntityAction` is a super-set of the _NgRx `Action`_.
+It has additional properties that guide NgRx Data's handling of the action. Here's the full interface.
+
+<code-example header="EntityAction">
+export interface EntityAction<P = any> extends Action {
```

Comment:
This example block isn't closed correctly.
I haven't verified it but I believe you have to use:
```suggestion
export interface EntityAction&lt;P = any&gt; extends Action {
```

On the other hand, the other code-examples seems to be fine...

![image](https://user-images.githubusercontent.com/28659384/57322867-df334d80-7104-11e9-9c9c-46867eeda749.png)

Reply:
Good catch. Thanks.